wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,636 @@
+---
+tags:
+  - UK Community Cup 7
```

Comment:
the title doesn't need to be repeated in the tags (every part of the title basically acts like a tag)
```suggestion
```